app.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "30d2e4bf-2402-4197-b844-e7082b07dc65",
-    "version": "0.0.3",
+    "version": "0.1.3",
```

Comment:
Are we sure about this version bump? I think we'd either need to go `0.0.4` or `0.1.0` as per semantic versioning standard

Reply:
Oops, you are right. 0.1.0

Reply:
```suggestion
    "version": "0.1.0",
```